elizhahn/niche-audience
# Styling/responsive design

## What does this PR do (summary of changes)?
- This adds responsive styling to the entire application
- The landing page, main dashboard, and remove shelf modal views were addressed and are fully responsive mobile to large desktop
## How should it be tested?
- For each of the listed views above, use responsive dev tool to shrink the application down and up to see that the layout changes at appropriate break points and UX is consistent
## Future Iterations:
- responsive styling for the error messages when adding a form
